components/DocsHelp.tsx

```diff
@@ -250,7 +251,7 @@ export function DocsHelp() {
               target='_blank'
               rel='noreferrer'
               className='px-[16px] py-[8px] cursor-pointer border-solid border-[#aaaaaa] border rounded-md hover:bg-gray-200'
-              href='https://github.com/orgs/json-schema-org/projects/16'
+              href={`https://github.com/json-schema-org/website/blob/main/pages/${path}.page.tsx`}
```

Comment:
Here, I am not sure why we are taking the user to the component that renders the page instead of the markdown that contains the data. If they want to edit some content on the page, wouldn't it make more sense to take them to the markdown content and not the component that renders it?

Reply:
That will be great. In the cases when the md exists pointing to the markdown will be the best destination.

Reply:
That's idea is fantastic thanks :) , I will get right on it.

Reply:
hey @benjagm, its a good idea to point to the existing md files, but since all md files don't have consistent names like some are called read_me.md , index.md and so on. How do you suggest i should attempt to solve this?